public/js/console-game.js

```diff
@@ -264,6 +278,6 @@ var Game;
     basicActions.go   = basicActions.move   ;
     basicActions.take = basicActions.pickup ;
     
-    this.moveTo("entry");
+    return this.moveTo("entry");
   }
 })();
```

Comment:
Oh god i just realized i had mostly browserified it but never merged. My bad. I'll have to make a careful merge into that branch later.